src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,8 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-
+    public static final Prefix PREFIX_CODE = new Prefix("m/");
+    public static final Prefix PREFIX_TYPE = new Prefix("n/");
```

Comment:
Just curious. Is there a need for `PREFIX_TYPE` when there is already a `PREFIX_NAME`? OR this is just for clarity when reading code?

Reply:
Yeah, this is for clarity (and just in case we want to have different keywords for them later on).

Reply:
Ok